build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -8,7 +8,6 @@
     shell:
       chdir: "{{ base_dir }}/working/smart-gateway-operator/build"
       cmd: |
-        OPERATOR_SDK="{{ base_dir }}/working/operator-sdk" \
```

Comment:
You might run into issues if you use the existing operator-sdk in the environment, which is likely to be operator-sdk-1.26.0 or something. Our system is still on the pre-1.x release of operator-sdk, so the commands may not play nicely.

Reply:
I'll take what's in the github action and download the same version, which is 0.19.4

Reply:
Actually, I think the issue is with build/stf-run-ci/tasks/main.yml 

I can update the "Get new operator sdk" task to make sure operator-sdk is in the expected location, using chdir: "{{ base_dir }}" like I have for the other tasks